README.md

```diff
@@ -53,5 +53,24 @@ open localhost:1313
 ### How to add new episode
```

Comment:
I would put this into a new file `RELEASE.md` or maybe github has a good way to manage that.
it would be nice to create a release with a tag for every new episode

Reply:
We could do that (RELEASE.md). But what are tags for? We are running on the latest anyway, don't we? I mean why would we want to go back?

Reply:
Good point